### ISSUE TITLE
How to use Multi-Head attention in local_attention?

### ISSUE THREAD

**Issue by JonasLi-19** [lucidrains/local-attention#28]
This is my code:
        src_att = self.local_attention(q, k, v, mask=None)
 And I am wondering how to specify the multi-head attention here?

**Comment by lucidrains** [comment]
@JonasLi-19 hey, you can use the [`LocalMHA`](https://github.com/lucidrains/local-attention/blob/master/local_attention/transformer.py#L48) class

**Comment by JonasLi-19** [comment]
Thanks! Iwill check the LocalMHA later.
Now I want to ask that if my code is as follows, **it doesn't have MultiHead-Attention inside?** 
(this contains one local-attention layer and six global-attention layers)

```
from local_attention import LocalAttention
...
...
class TransformerRegressor(nn.Module):
    def __init__(self, input_dim, seq_l, dim_feedforward, nhead=8, num_layers=6, local_attn_window_size=64):
        super(TransformerRegressor, self).__init__()
        self.input_dim = input_dim
        self.seq_l = seq_l
        self.pos_enc = ...

        self.q_proj = nn.Linear(input_dim, input_dim)
        self.k_proj = nn.Linear(input_dim, input_dim)
        self.v_proj = nn.Linear(input_dim, input_dim)

        # LocalAttention
        self.local_attention = LocalAttention(
            dim=input_dim,
            window_size=local_attn_window_size,
            causal=False,
            autopad = True,
            look_backward=1,
            look_forward=1,
            dropout=dropout_rate
        )
    ....
            encoder_layers =...

    def forward(self, src):
        # encode position
        ...

        # the Q K V in local-attention layer
        q = self.q_proj(src)
        k = self.k_proj(src)
        v = self.v_proj(src)
        
        # LocalAttention + resnet + LayerNorm
        src_att = self.local_attention(q, k, v, mask=None)
        src = self.layer_norm1(src + src_att)
        
        residual = src

        # TransformerIntermediate
        src = self.intermediate_layer(src)
        src = self.layer_norm2(residual + src)

        # TransformerEncoder
        ...
        for encoder_layer in self.transformer_encoder:
            src = encoder_layer(src)

        output = ...
```

**Comment by lucidrains** [comment]
@JonasLi-19 `LocalMHA` would be the full multihead attention class

`LocalAttention` only contains the localized attention operator, you'll have to pass in the queries, keys, values already split into multiheads
